### PR TITLE
[front] dev: dustapps sync script

### PR DESCRIPTION
**Pull request by tdraier**
## Description

Script to synchronize dust apps and execute in dev environments

### CODE REVIEW

File: front/admin/copy_apps.sh [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+fi
+
+mkdir -p /tmp/dust-apps
+
+cd ${DIR}/..
+
+./admin/cli.sh registry dump > /tmp/dust-apps/specs 2> /dev/null
+
+REGISTRY_COUNT=$(cat /tmp/dust-apps/specs | wc -w)
+IN_CLAUSE=$(cat /tmp/dust-apps/specs |  cut -f4 -d\|  | sed -E "s/(.*)/'\\1'/g" | paste -sd "," - )
```
Comment (flvndvd) [inline front/admin/copy_apps.sh:58]:
```suggestion
IN_CLAUSE=$(cat /tmp/dust-apps/specs | cut -f4 -d\| | sed -E "s/(.*)/'\\1'/g" | paste -sd "," - )
```

File: front/admin/cli.ts [hunk 2]
```diff
@@ -560,6 +567,24 @@ const transcripts = async (command: string, args: parseArgs.ParsedArgs) => {
   }
 };
 
+const registry = async (command: string, args: parseArgs.ParsedArgs) => {
+  switch (command) {
+    case "dump": {
+      Object.keys(DustProdActionRegistry).forEach((key) => {
```
Comment (flvndvd) [inline front/admin/cli.ts:573]: Let's add a comment to prevent someone from breaking the format below 🙏

**Review by flvndvd — APPROVED** [review]

File: front/admin/cli.ts (hunk 2 above)
Comment (flvndvd) [reply front/admin/cli.ts:573]: Or if we can use `jq`

Comment (tdraier) [reply front/admin/cli.ts]: let's use jq

File: front/admin/copy_apps.sh [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    exit 0
+fi
+
+mkdir -p /tmp/dust-apps
+
+cd ${DIR}/..
+
+./admin/cli.sh registry dump > /tmp/dust-apps/specs 2> /dev/null
+REGISTRY_COUNT=$(cat /tmp/dust-apps/specs | jq -r '[.[].app.appHash] | join("\n")' | wc -l)
+IN_CLAUSE=$(cat /tmp/dust-apps/specs | jq -r '[.[].app.appHash] | map("'"'"'" + . + "'"'"'") | join(",")' )
```
Comment (flvndvd) [inline front/admin/copy_apps.sh:57]: 🤯 
Can we do and add a comment 
```
# Reads appHash values from JSON, escapes them for shell usage, and concatenates them with commas for SQL queries.
IN_CLAUSE=$(jq -r '[.[].app.appHash] | map("\(. | @sh)") | join(",")' /tmp/dust-apps/specs)
```

**Comment by Fraggle** [comment]
@tdraier no mergeable ?

**Review by flvndvd — APPROVED** [review]
Works great with the latest changes.

**Comment by tdraier** [comment]
@Fraggle merged, feel free to test it :-)

**Pull request merged**